_posts/2015-04-02-welcome-to-sittercity-tech.markdown

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title:  "Welcome to Sittercity Technology!"
+date:   2015-04-02 09:00:00
+categories: sittercity
+---
+
+Welcome! This here is our new blog! As is fitting of a new journal let's start ourselves off with an introduction to Sittercity Tech.
+
+####  What we do
+
+So! [Sittercity](https://www.sittercity.com). What do we do? Sittercity is a service that connects families and sitters. Our goal is to
```

Comment:
I think we need to get help from marketing for this first paragraph. We need to make sure that what we say here(What we do) aligns with the message that the company is putting out there.